readme.md

```diff
@@ -52,8 +54,10 @@ body{
   background-color = "red";
 }
 ```
-
-> Your answer...
+>The declaration block for this css selector requires a colon instead of the equal sign between the property name and value assigned to it.
+body {
+background-color: "red";
+}
```

Comment:
:+1: 
I think most browsers will accept quotes around the color name, but I'd note that it's better for color names not to use quotes.